NonDecreasingSequence.java

```diff
@@ -0,0 +1,30 @@
+import java.util.Scanner;
+
+class NonDecreasingSequence {
+  public static void main(String[] args) {
+    System.out.println("Enter a sequence of integers separated by spaces.");  
+    Scanner input = new Scanner(System.in);
+    String sequence = input.nextLine();
+    String sequenceSplit[] = sequence.split(" ");   
+    boolean isNonDecreasing = true;   
```

Comment:
Find solution without using any flags that shows whether sequence decreasing or not